version/distro/distro.go

```diff
@@ -31,7 +31,7 @@ const (
 	WDMyCloud = Distro("wdmycloud")
 	Unraid    = Distro("unraid")
 	Alpine    = Distro("alpine")
-	UDMPro    = Distro("udmpro")
+	Ubnt      = Distro("ubnt") // Ubiquiti Networks
```

Comment:
It's not quite an acronym but I think UBNT is better than Ubnt.

Reply:
Updated.